django/db/migrations/graph.py

```diff
@@ -126,33 +127,59 @@ def clear_cache(self):
                 self.node_map[node].__dict__.pop('_descendants', None)
             self.cached = False
 
-    def forwards_plan(self, node):
+    def forwards_plan(self, root):
```

Comment:
I'd rename that to "target". See `root_nodes()` for an explanation of what we consider root nodes.